### PR TITLE
Cohort 3 base types and intro to OOP

### PR DESCRIPTION
**Pull request by MeganStodel**
Slides and updated README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +160,9 @@ Chapter 11 - Function operators
 - Cohort 2: [Kevin Kent](https://twitter.com/kevin_m_kent) (SLIDES | [video](https://youtu.be/fD1QJB2pHik))
 - Cohort 3: [Anna-Leigh Brown](https://twitter.com/annaleighbrown2) (SLIDES) | [video](https://youtu.be/fW7Di01gLhw))
 
+Chapter 12 - OOP base types
+- Cohort 3: [Megn Stodel](https://twitter.com/MeganStodel) ([slides](https://github.com/MeganStodel/bookclub-Advanced_R/tree/cohort-3-base-types/Presentations/Week12/Cohort3))
```
Comment (tanho63) [inline README.md:164]: Just curious if you'd meant to sign your name like that :)

Comment (jonthegeek) [inline README.md:164]:
```suggestion
- Cohort 3: [Megn Stodel](https://twitter.com/MeganStodel) ([slides](https://github.com/MeganStodel/bookclub-Advanced_R/tree/cohort-3-base-types/Presentations/Week12/Cohort3) | [video](https://youtu.be/IL6iJhAsZAY))
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
@tanho63 The folder should be `Cohort03`, right?

**Comment by tanho63** [comment]
> @tanho63 The folder should be `Cohort03`, right?

Um, I don’t think so (unless you want to take on the big renaming PR)

**Comment by jonthegeek** [comment]
> > The folder should be `Cohort03`, right?
> 
> Um, I don’t think so (unless you want to take on the big renaming PR) @jonthegeek

Ok, if we aren't doing that here already, we won't worry about it yet :)

File: README.md (hunk 1 above)
Comment (MeganStodel) [reply README.md]: Ugh...thanks! I've changed it now.

File: README.md [hunk 2]
```diff
@@ -160,6 +160,9 @@ Chapter 11 - Function operators
 - Cohort 2: [Kevin Kent](https://twitter.com/kevin_m_kent) (SLIDES | [video](https://youtu.be/fD1QJB2pHik))
 - Cohort 3: [Anna-Leigh Brown](https://twitter.com/annaleighbrown2) (SLIDES) | [video](https://youtu.be/fW7Di01gLhw))
 
+Chapter 12 - OOP base types
+- Cohort 3: [Megan Stodel](https://twitter.com/MeganStodel) ([slides](https://github.com/MeganStodel/bookclub-Advanced_R/tree/cohort-3-base-types/Presentations/Week12/Cohort3))
```
Comment (tanho63) [inline README.md:164]:
```suggestion
- Cohort 3: [Megan Stodel](https://twitter.com/MeganStodel) ([slides](https://github.com/MeganStodel/bookclub-Advanced_R/tree/cohort-3-base-types/Presentations/Week12/Cohort3)  | [video](https://youtu.be/IL6iJhAsZAY))
```

**Review by tanho63 — APPROVED** [review]
LGTM!

**Pull request merged**